cfdm/test/test_DimensionCoordinate.py

```diff
@@ -107,6 +108,21 @@ def test_DimensionCoordinate_climatology(self):
         self.assertTrue(x.del_climatology())
         self.assertIsNone(x.del_climatology(None))
 
+    def test_DimensiconCoordinate_array(self):
+        """Test the `DimensionCoordinate.datetime_array` method."""
+        f = cfdm.example_field(0)
+        t = f.construct("time")
+        self.assertEqual(t.array, 31)
```

Comment:
For thoroughness, can we also include a check that these direct methods are effectively aliases for the equivalent `data.` method, i.e:

```suggestion
        self.assertEqual(t.array, 31)
        self.assertEqual(t.array, t.data.array)
```
?